### PR TITLE
chore: add hx to about

### PR DESCRIPTION
**Pull request by Etherite1**
as per popular request LOL

### CODE REVIEW

File: frontend/src/pages/About.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import eh from '../images/headshots/evan-hu.jpg';
 import hl from '../images/headshots/hao-li.jpg';
 import hy from '../images/headshots/harry-yu.jpg';
 import hs from '../images/headshots/harshal-sheth.jpg';
+import hx from '../images/headshots/humphrey-xu.jpg'
```
Comment (Josh-Cena) [inline frontend/src/pages/About.tsx:19]:
```suggestion
import hx from '../images/headshots/humphrey-xu.jpg';
```

**Comment by Josh-Cena** [comment]
@Etherite1 Could you run `bun checks:fix` locally

File: api/src/config.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ const port = process.env.FRONT_END_PORT ?? 3000;
 // Frontend server endpoint (used for redirects)
 export const FRONTEND_ENDPOINT = isDev
   ? `https://localhost:${port}`
-  : (process.env.FRONTEND_ENDPOINT ?? 'https://coursetable.com');
+  : process.env.FRONTEND_ENDPOINT ?? 'https://coursetable.com';
```
Comment (Josh-Cena) [inline api/src/config.ts:46]:
```suggestion
  : (process.env.FRONTEND_ENDPOINT ?? 'https://coursetable.com');
```

**Review by Josh-Cena — COMMENTED** [review]
FYI you probably have a different version of Prettier installed. Perhaps running bun install would help.

**Comment by Josh-Cena** [comment]
Wait your bun checks did not resize the image?? That's weird

**Comment by bearsyankees** [comment]
> Wait your bun checks did not resize the image?? That's weird

i just tried too and no dice

**Review by Josh-Cena — APPROVED** [review]
Works finally

Will investigate `bun checks` later

**Pull request merged**